addon/styles/partials/_uni-tooltip.scss

```diff
@@ -0,0 +1,8 @@
+.uni-tooltip {
+  @include tooltip();
+  display: inline-block;
```

Comment:
Are we going to force this to always be display: inline-block or should we delegate this problem to the platforms?